lib/rspec/shell/expectations/stubbed_call.rb

```diff
@@ -41,6 +41,10 @@ def called_with_args?(*args, position: false)
           @call_log.called_with_args?(*args, sub_command_series: @args, position: position)
         end
 
+        def get_argument_count(*arg)
+          @call_log.get_argument_count(*arg)
```

Comment:
This will need sub command list sent in, or it won't correctly match commands that have been created using .with_args correctly. It's another goofy side effect of the design